dialogflow/cloud-client/create_document_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import datetime
+import os
+
+import dialogflow_v2beta1 as dialogflow
+import pytest
+
+import document_management
+
+PROJECT_ID = os.getenv('GCLOUD_PROJECT')
+KNOWLEDGE_BASE_NAME = 'knowledge_' \
+                      + datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+DOCUMENT_DISPLAY_NAME = 'test_document_' \
+                     + datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+pytest.KNOWLEDGE_BASE_ID = None
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
@gguuss, while I wait for the quota increase. 

How do you feel about these types of tests? 

I was gonna modify the rest and separate them into individual files in different PRs, but the current Knowledge and Document tests were checking that only 1 thing existing, so I at least had to update those for the 4 python versions.

Reply:
I'm fine for checking that all the response parameters are there but adding more detailed checks, particularly in ML APIs, can make the checks more brittle.